zeus/reports.py

```diff
@@ -244,7 +244,7 @@ def csv_from_polls(election, polls, lang, outfile=None):
         try:
             outfile.seek(0)
             return outfile.read()
-        except:
+        except ValueError:
```

Comment:
This is fishy. Is anything using the results of this function? I think it would be better if this function just wrote results to a file, then whoever is calling it, will handle reading results back if they want to.